retromock/RELEASE.md

```diff
@@ -5,11 +5,11 @@
 Be sure that [build.gradle](build.gradle), [Changelog](../CHANGELOG.md) and [Readme](../README.md) are updated and contain new, to be released version.
 
 ### Run static checkers and tests
-`./gradlew clean build checkstyle pmd findbugs test`
+`./gradlew clean build checkstyleMain checkstyleTest pmdMain pmdTest findbugsMain findbugsTest test`
 
 ### Run gradle task
 
-`./gradlew clean build javadocs jar sourceJar`
+`./gradlew clean build javadoc jar sourceJar`
```

Comment:
Are you sure we don't need javadocJar jere? I believe we need a jar with javadoc to upload to Bintray

Reply:
My bad, it seems as bintray task executes javadocJar